plugin/src/modutils.ml

```diff
@@ -0,0 +1,97 @@
+open Util
+open Names
+open Globnames
+open Declarations
+open Indutils
+
+(*
+ * Pull any functor parameters off the module signature, returning the list of
+ * functor parameters and the list of module elements (i.e., fields).
+ *)
+let decompose_module_signature mod_sign =
```

Comment:
Was this elsewhere before? I think I remember seeing this in the past

Reply:
Aha, you're right -- the exact function appears [here](https://github.com/uwplse/coq-plugin-lib/blob/master/src/coq/logicutils/transformation/transform.ml#L110) in the plugin library, though it isn't exported. Per your [other comment](https://github.com/uwplse/ornamental-search/pull/76#discussion_r369830606) (second below), refactoring `modutils.ml` into the plugin library will deduplicate code like this.